packages/tenderly-hardhat/package.json

```diff
@@ -23,6 +23,9 @@
     "LICENSE",
     "README.md"
   ],
+  "bin": {
+    "tenderly-hardhat-initt": "dist/scripts/hardhat-tenderly.js"
```

Comment:
`init` or `initt`? 

...not sure why I even opened this PR, but now that I have I had to ask 😅